### PR TITLE
Ignore dotfiles by default

### LINKED ISSUE #14: Ignore dot-files when creating DNA bundle

### PR DESCRIPTION
**Pull request by sphinxc0re**
closes #14

### CODE REVIEW

**Comment by lucksus** [comment]
That was fast! I just created the ticket, got my tea from the kitchen and there is already a PR for it.
Cheers! :D

**Comment by lucksus** [comment]
It seems that dot-files are being ignored, but not dot-directories, like .git for instance...

**Comment by lucksus** [comment]
Cool, it works! Just missing documentation ;)

**Comment by lucksus** [comment]
Please review, need merge. Does ignore dot-files and files mentioned in .hcdevignore. app-spec-rust already has a .hcdevignore file.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,18 @@ So in a given Zome we have two things:
 
 In order for Holochain to run your app, you have to build your code into a single packaged file. Those instructions follow.
 
+## Ignoring files
+
+Sometimes, you'll want to exclude files in your project directory to get a straight .hcpkg file that can be understood by Holochain. In oder to do that, just create a `.hcdevignore` file. It has a simmilar structure to `.gitignore` files:
```
Comment (Connoropolous) [inline README.md]: Spelling -> In "order"
And "similar"

File: README.md [hunk 2]
```diff
@@ -62,6 +62,18 @@ So in a given Zome we have two things:
 
 In order for Holochain to run your app, you have to build your code into a single packaged file. Those instructions follow.
 
+## Ignoring files
+
+Sometimes, you'll want to exclude files in your project directory to get a straight .hcpkg file that can be understood by Holochain. In order to do that, just create a `.hcdevignore` file. It has a similar structure to `.gitignore` files:
```
Comment (thedavidmeister) [inline README.md]: at the risk of being "that guy" and bike shedding this to death... what do you think of just `.hcignore`?

Comment (sphinxc0re) [reply README.md]: Also the tool also recognizes `.ignore` files, so we could do just tha

Comment (zippy) [reply README.md]: Lets use `.hcignore` rather than `.hcdevignore` because we know the command name will change.

**Pull request merged**